code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -251,6 +251,17 @@ var/global/list/datum/stack_recipe/cardboard_recipes = list (
 	new/datum/stack_recipe("cardboard cutout", /obj/item/cardboard_cutout, 5),
 )
 
+/obj/item/stack/sheet/cardboard/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/stamp/clown) && !istype(loc, /obj/item/storage))
+		var/atom/droploc = drop_location()
+		if(use(TRUE))
+			playsound(I, 'sound/items/bikehorn.ogg', 50, 1, -1)
+			to_chat(user, "<span class='notice'>You stamp the cardboard! It's a clown box! Honk!</span>")
+			if(amount >= 0)
```

Comment:
If I recall correctly such a check is already done in the stack's logic. In addition, why would this also spawn when the amount is 0?

If somehow 0 cardboard sheet is being made it is a problem on the stack's logic and shouldn't be fixed this way.

Reply:
You recall correctly the stak 's logic  was already doing the job.
Usually if i see the word //bugfix i dont question why its there, my bad